extension/src/experiments/workspace.ts

```diff
@@ -250,7 +250,8 @@ export class WorkspaceExperiments extends BaseWorkspaceWebviews<
     if (!input) {
       return
     }
-    return runCommand(...args, input)
+    await runCommand(...args, input)
+    return input
```

Comment:
Updated `getInputAndRun` to return the input instead of the command result. I checked commands that use this util and they aren't using the result of the command anywhere.

Reply:
[Q] Can you return both? In this instance, we know that `runCommand` returns a string. I'd want to check if the response from `dvc` was non-empty before calling for updates.

Reply:
`getInputAndRun` no longer needs to be updated since we no longer need the `WebviewMessages` to have the input.